### DISCUSSION TITLE
High Resolution GeoTIF Maps

### DISCUSSION THREAD

**Discussion by Warren-Whitfield** [pelicanmapping/osgearth#2592] [Support Q&A] [answered]
Where can I obtain high resolution maps similar to 'world.tif' in the samples.
I typically don't have more than a maximum of 200km radius in my display. In fact I limit anything more than that. This more or less equates to a  max camera height of 780km in my system. When the user zooms in I need the map to be much more detailed. Currently its just a blur.
I assume that this type of data will have to be purchased though, but we would need to start looking at that anyway.

**Comment by plevy** [comment, marked as answer]
Yes, you can certainly purchase imagery from resellers such as 
https://apollomapping.com/
https://www.arlula.com/pricing/
https://skywatch.com/data-pricing/

We have a dataset called ReadyMap tiles 
https://www.pelicanmapping.com/readymap-data

or you can look for government owned imagery or state owned in the US.
https://earthexplorer.usgs.gov/
